tests/widgets_paragraph.rs

```diff
@@ -79,6 +79,79 @@ fn widgets_paragraph_can_wrap_its_content() {
     );
 }
 
+#[test]
+fn widgets_paragraph_can_align_spans() {
+    let right_s = "The library is based on the principle of immediate rendering.";
+    let default_s = "This should be aligned to paragraph.";
```

Comment:
Perhaps change these strings to unambiguously self document the code. E.g. "This string will override the paragraph alignment to be right aligned" and "This string will be aligned based on the alignment of the paragraph"

Reply:
Good idea!